containers/vm-provisioning/config

```diff
@@ -0,0 +1,24 @@
+# this file is sourced by several scripts in this directory
+
+env=unknown
+if test -f instance; then
+    env=`cat instance`
+fi
+
+declare -A \
+    aws_image \
+    aws_instance_type \
+    aws_spot_max_price \
+
+image=b8a0e7a5-fa58-4e2b-9fa5-e2c95159d3a2
```

Comment:
Might add comments on where to get these image and ami's? or more information on what exactly they are?